detect-nuget-inspector/detect-nuget-inspector/Inspection/Util/InspectorUtil.cs

```diff
@@ -61,5 +62,30 @@ public static string GetProjectAssemblyVersion(string projectDirectory)
 
             return null;
         }
+        
+        public static string GetAttributeInformation(XmlAttributeCollection attributes, string checkString, XmlNode package)
+        {
+            string attributeStr = null;
+            
+            XmlAttribute attribute = attributes[checkString];
+
+            if (attribute == null)
```

Comment:
Should attribute.Value also be validated for null or blank string?

Reply:
We are already making those validations in the respective parser class so we should be good there.